### PR TITLE
Mes 8892 mobile accessibility

### PR DESCRIPTION
**Pull request by RLCorp**
## Description
moved the MobileAccessibility functionality to a new plugin
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/app/__tests__/app.component.spec.ts [hunk 1]
```diff
@@ -33,6 +33,8 @@ import { DateTimeProviderMock } from '@providers/date-time/__mocks__/date-time.m
 
 import { Subscription } from 'rxjs';
 import { SideMenuClosed, SideMenuItemSelected, SideMenuOpened } from '@pages/dashboard/dashboard.actions';
+import { AccessibilityService } from '@providers/accessibility/accessibility.service';
+import { AccessibilityServiceMock } from '@app/__mocks__/accessibility.service.mock';
```
Comment (matthew2564) [inline src/app/__tests__/app.component.spec.ts]: Can we move this mock into the same folder as the service, others would typicall follow this structure

```
/provider/
   /__tests__
   /__mocks__
provider.ts   
```

File: src/app/pages/waiting-room/components/manoeuvres-pass-cert/__tests__/manoeuvres-pass-cert.spec.ts [hunk 2]
```diff
@@ -20,6 +21,7 @@ describe('ManoeuvresPassCertificateComponent', () => {
       ],
       providers: [
         PassCertificateValidationProvider,
+        MobileAccessibility,
```
Comment (matthew2564) [inline src/app/pages/waiting-room/components/manoeuvres-pass-cert/__tests__/manoeuvres-pass-cert.spec.ts]: Can this be replaced with the accessibility service

File: src/app/providers/accessibility/__test__/accessibility.service.spec.ts [hunk 3]
```diff
@@ -10,6 +11,7 @@ describe('AccessibilityService', () => {
     TestBed.configureTestingModule({
       providers: [
         AccessibilityService,
+        MobileAccessibility,
```
Comment (matthew2564) [inline src/app/providers/accessibility/__test__/accessibility.service.spec.ts:15]: We should now have the ability to add proper unit testing of all methods in the `AccessibilityService` seeing as we can now hook into this

Comment (RLCorp) [reply src/app/providers/accessibility/__test__/accessibility.service.spec.ts:15]: increased coverage in newest commit

File: src/app/pages/waiting-room/components/manoeuvres-pass-cert/__tests__/manoeuvres-pass-cert.spec.ts (hunk 2 above)
Comment (RLCorp) [reply src/app/pages/waiting-room/components/manoeuvres-pass-cert/__tests__/manoeuvres-pass-cert.spec.ts]: fixed in newest commit

File: src/app/__tests__/app.component.spec.ts (hunk 1 above)
Comment (RLCorp) [reply src/app/__tests__/app.component.spec.ts]: fixed in newest commit

**Review by matthew2564 — APPROVED** [review]

**Review by gittins7 — APPROVED** [review]

**Pull request merged**